### DISCUSSION TITLE
이영우

### DISCUSSION THREAD

**Discussion by z1nun** [z1nun/ASTERISK-#15] [독서일지]
# 각자 선정한 개발자 필독서를 읽고 그날 읽은 내용을 간단히 정리해서 적어주세요!

## Clean Code

- 1일차 8p ~ 14p
- 2일차 15p  ~ 21p
-

**Comment by 2Zerozero** [comment]
1일차 8p ~ 14p
깨끗한 코드는 테스트 케이스가 존재하는 코드이며, 문학적인 코드여야 한다.
깨끗한 코드란 작성자가 아닌 다른사람들도 쉽게 읽을수 있으며, 고칠수 있는 코드여야한다.

**Comment by 2Zerozero** [comment]
2일차 15p ~ 21p
프로그래머란, 프로그래밍 언어를 단순하게 보이도록 만드는 열쇠 같은 존재다.
깨끗한 코드란, 예상한 대로 작동하는 코드이며, 이런 코드를 짜기위해 정성을 들여서 코드를 작성해야한다.
